src/PowerShellEditorServices.Host/CodeLens/PesterCodeLensProvider.cs

```diff
@@ -54,7 +54,11 @@ public PesterCodeLensProvider(EditorSession editorSession)
                     new ClientCommand(
                         "PowerShell.RunPesterTests",
                         "Run tests",
-                        new object[] { scriptFile.ClientFilePath, false /* No debug */, pesterSymbol.TestName })),
+                        new object[] {
+                            scriptFile.ClientFilePath,
+                            false /* No debug */,
+                            pesterSymbol.TestName,
+                            pesterSymbol.ScriptRegion.StartLineNumber })),
```

Comment:
Can ScriptRegion ever be null? Given that we get the PesterSymbolReference from the AST, I don't think so. We could make this `pesterSymbol.ScriptRegion?.StartLineNumber ?? null` but I'm not sure that is necessary.

Reply:
There's nothing in the immediate constructor to prevent it being null. It feels like we shouldn't need it, but having it crash would be much worse than just the null check.

The `?? null` shouldn't be needed in that case I don't think -- the `?.StartLineNumber` should resolve to null by itself.

Reply:
Duh.  Good point.  :-)  I'll make the change just to be on the safe side.